testutil/x/staking/proto/cosmos/staking/v1beta1/mtx.proto

```diff
@@ -0,0 +1,96 @@
+syntax = "proto3";
+package cosmos.staking.v1beta1;
+
+import "google/protobuf/any.proto";
+import "google/protobuf/timestamp.proto";
+import "gogoproto/gogo.proto";
+
+import "cosmos_proto/cosmos.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "cosmos/staking/v1beta1/mstaking.proto";
+import "cosmos/msg/v1/msg.proto";
+import "amino/amino.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/testutil/x/staking/types";
+
+// Msg defines the staking Msg service.
+service Msg {
+  option (cosmos.msg.v1.service) = true;
+
+  // CreateValidator defines a method for creating a new validator.
+  rpc CreateValidator(MsgCreateValidator) returns (MsgCreateValidatorResponse);
+
+  // Delegate defines a method for performing a delegation of coins
+  // from a delegator to a validator.
+  rpc Delegate(MsgDelegate) returns (MsgDelegateResponse);
+
+  // UpdateParams defines an operation for updating the x/staking module
+  // parameters.
+  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse) {
+    option (cosmos_proto.method_added_in) = "cosmos-sdk 0.47";
+  }
```

Comment:
we could remove this, make it super simple, just creation of validator. If they want delegations they would use the staking module